slices/openssh-server.yaml

```diff
@@ -0,0 +1,55 @@
+package: openssh-server
+
+slices:
+  base:
+    essential:
+      - openssh-server_bins
+      - openssh-server_config
+      - openssh-server_services
+
+  bins:
+    # openssh-server actually depends on adduser as well to create and remove
+    # a user during package install/removal - however we don't support this
+    # currently so adduser is not added here.
+    essential:
+      - init-system-helpers_bins
+      - libaudit1_libs
+      - libc6_libs
+      - libcom-err2_libs
+      - libcrypt1_libs
+      - libgssapi-krb5-2_libs
+      - libkrb5-3_libs
+      - libpam-modules_libs
+      - libpam-runtime_base
+      - libpam0g_libs
+      - libselinux1_libs
+      - libssl3_libs
+      - libsystemd0_libs
+      - libwrap0_libs
+      - sysvinit-utils_base
```

Comment:
Do we need this one? I only noticed now that this is not listed as a dependency in the package control file.

Reply:
You are right, seems this has changed